app/components/ui/atoms/PersonPod.js

```diff
@@ -0,0 +1,117 @@
+import React from 'react'
+import styled from 'styled-components'
+import { th } from '@pubsweet/ui-toolkit'
+import { Action, Button } from '@pubsweet/ui'
+import { Flex, Box } from 'grid-styled'
+
+import Icon from './Icon'
+
+const RegularP = styled.p`
+  font-size: ${th('fontSizeBase')};
+  line-height: ${th('fontLineHeightBase')};
+  margin: 0;
+`
+
+const SmallP = styled.p`
+  font-size: ${th('fontSizeBaseSmall')};
+  line-height: ${th('fontLineHeightBaseSmall')};
+  // vertical spacing of 6px comes from: 24px grid - fontLineHeightBaseSmall
+  margin: 3px 0;
+`
+
+const SmallAction = styled(Action)`
+  font-size: ${th('fontSizeBaseSmall')};
+  line-height: ${th('fontLineHeightBaseSmall')};
+  margin: 3px 0;
+`
+
+const StyledButton = styled(Button)`
+  height: 100%;
+  border: none;
+`
+
+const StyledPicker = styled(Flex)`
+  display: flex;
+  justify-content: space-between;
+
+  background: ${th('colorSecondary')};
+  border: ${th('borderWidth')} ${th('borderStyle')} ${th('colorBorder')};
+  border-radius: ${th('borderRadius')};
+  color: ${th('colorText')};
+  font-family: ${th('fontInterface')};
+  font-size: ${th('fontSizeBase')};
+  min-width: 312px;
+  height: 128px;
```

Comment:
Where do these values come from?

Reply:
Guesswork. But I've just seen Guiliano has replied in Slack - https://elifesciences.slack.com/archives/C6LKMFK5Z/p1531910953000087

Reply:
Makes sense for height but does it need a min-width? Seems like it is likely to cause issues in some as yet unforeseen responsive layouts.